openmeetings-webservice/pom.xml

```diff
@@ -138,6 +138,11 @@
 			<groupId>org.codehaus.jettison</groupId>
 			<artifactId>jettison</artifactId>
 		</dependency>
+		<dependency>
+			<groupId>io.swagger.core.v3</groupId>
+			<artifactId>swagger-annotations</artifactId>
+			<version>2.1.10</version>
```

Comment:
could you move version to properties (and generalize where possible)
so i can easier check for updates? :)